index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="/style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
+</head>
+
+<body>
+    <nav>
+        <img src="https://pmcvariety.files.wordpress.com/2017/09/youtube-logo.png?w=896" width="200" height="100" alt="" width="200" height="100">
+        <input class="searchbar" type="search">
+        <input class="searchbutton" type="button" value="search">
+        <div class="right">
+            <img src="https://via.placeholder.com/35x35/fff" alt="">
+            <img src="https://via.placeholder.com/35x35/000" alt="">
+            <img src="https://via.placeholder.com/35x35/fff000" alt="">
+        </div>
+    </nav>
+
+    <section id="content">
+        <div id="main_player">
+            <iframe width="98%" height="550" src="https://www.youtube.com/embed/ZnuwB35GYMY" frameborder="0"
+                allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+                allowfullscreen></iframe>
+            <h2>How To Embed A YouTube Video</h2>
+            <div class="meta_data">
+                <p id="views">12344 views</p>
+
+                <i class="material-icons c_icons">thumb_up</i>
+                <i class="material-icons  c_icons">thumb_down</i>
+                <i class="material-icons c_icons">share</i>
+            </div>
+
+            <div class="user">
+                <div id="avatar_container">
```

Comment:
![vids](https://user-images.githubusercontent.com/31304943/69574936-acca3e80-0fc9-11ea-876c-e9be6b6954e9.png)

From the image above, the video section needs to be well aligned to the left-hand side so that you can fulfill the `Sidebar Recommended videos` requirements(more on this below).